opm/parser/share/keywords/000_Eclipse100/O/OPERNUM

```diff
@@ -0,0 +1,5 @@
+{
+ "name" : "OPERNUM",
```

Comment:
I'm not quite sure what you mean.  If you read the Eclipse manual, it will become obvious that we don't _support_ `OPERNUM`.  The purpose of this PR was to not _die_ when we encountered this keyword in a very special model I have on my desk.

If someone comes with a model using `OPERNUM` properly, we must consider actually implementing the proper support, but that's a non-trivial job.